hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AuthenticationBaseMessageTask.java

```diff
@@ -126,14 +127,24 @@ private AuthenticationStatus authenticate() {
 
     private AuthenticationStatus authenticate(SecurityContext securityContext) {
         Connection connection = endpoint.getConnection();
+        Boolean passed = Boolean.FALSE;
```

Comment:
What is the added value of a Boolean compared to boolean?

Reply:
It avoids boxing in the `addParameter` call.